rules/do-you-know-how-to-brand-your-api-portal/rule.md

```diff
@@ -1,25 +1,33 @@
 ---
 type: rule
 archivedreason: 
-title: Comments - Do you know how to brand your API portal?
+title: Do you know how to brand your API portal?
 guid: 2ca8fb8c-f799-4b66-a0dd-b9cb5f0ad3f0
 uri: do-you-know-how-to-brand-your-api-portal
 created: 2021-05-11T22:00:01.0000000Z
 authors:
+- title: William Liebenberg
+  url: https://ssw.com.au/people/william-liebenberg
 - title: Bryden Oliver
   url: https://ssw.com.au/people/bryden-oliver
 related: 
 - do-you-know-the-best-tools-to-manage-apis
 - do-you-know-when-to-mockup-your-apis
 redirects:
-- comments-do-you-know-how-to-brand-your-api-portal
```

Comment:
Is there a reason to remove this redirect?

Reply:
fixed